Ejemplo1.py

```diff
@@ -1,2 +1,2 @@
 for i in (range(20):  
```

Comment:
El código tiene un error en la declaración del for, favor de corregir por favor, gracias.